lib/Conversion/TTIRToTTMetal/TTIRToTTMetal.cpp

```diff
@@ -676,36 +676,158 @@ class TTIRToTTMetalDispatchRewriter : public OpRewritePattern<ttir::GenericOp> {
       builder.create<ttkernel::BinaryOpInitCommonOp>(
           op.getLoc(), cbOperands[operandIndices[0]],
           cbOperands[operandIndices[1]], cbOperands[numDpsInputs]);
-      builder.create<ttkernel::MulTilesInitOp>(op.getLoc(),
-                                               cbOperands[operandIndices[0]],
-                                               cbOperands[operandIndices[1]]);
+      builder.create<ttkernel::MulTilesInitFOp>(op.getLoc());
```

Comment:
Those are not the same. `mul_tiles_init_f` initializes only math, and `mul_tiles_init` initializes both unpack and math. I think we should even call `mul_tiles_init_f` in all other eltwise ops we have, at least I think it should be called when you look into https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/ttnn/operations/eltwise/binary/device/kernels/compute/eltwise_binary_kernel.cpp

I am not sure why this is called instead of regular `mul_tiles_init` but I will investigate. 

Point being none of these functions are deprecated, both are used